packages/dcos-integration-test/extra/test_calico_networking.py

```diff
@@ -0,0 +1,190 @@
+import contextlib
```

Comment:
Because we have too much code in test_networking.py covering a lot of scenarios, it's better to maintain in the future by having different code style and maintainers.

Reply:
I'll keep minimal change to add integration test for calico networking.